create-kafka-queues/create-topics.sh

```diff
@@ -68,13 +70,13 @@ else
     returnVal=$(./bin/kafka-topics.sh --list --zookeeper $ZOOKEEPER --topic ${!topic})
     if [ -z "${returnVal}" ]; then
       # Use variable indirection to get the contents of KAFKATOPIC e.g palisade 1 1
-      echo "Creating topic ${!topic}"
+      echo "Creating topic ${!topic} $(date)"
```

Comment:
Would it not be worth changing this to a `printf` like the others rather than just sticking the date value on the end?
```suggestion
      printf 'Creating topic %s - at %s' "${!topic}" "$(date)"
```